astropy/units/tests/test_equivalencies.py

```diff
@@ -215,6 +217,61 @@ def test_massenergy():
                        pow_eV.value, rtol=1e-7)
 
 
+@pytest.mark.parametrize(
+    'mass,length,time',
+    [[2 * u.solMass, 3 * u.km, 10 * u.microsecond],
+     [1 * u.kg, 7.54e-31 * u.km, 2.47e-36 * u.s],]
+)
+def test_mass_length_time_geometrized_equivalency(mass, length, time):
+    """Known cases of mass, length, time equivalencies. An example being
+    Schwarzschild radius for sun is 3 km, which is about 10 microsecond
+    of light crossing time.
+    """
+    converted_mass = mass.to('kg', equivalencies=u.geometrized('mass')).value
```

Comment:
Comparisons are normally done like
```
assert_quantity_allclose(mass.to('kg', equivalencies=u.geometrized('mass'), comparison, atol=...)
```
see elsewhere in this file.

Reply:
These particular tests do not actually seem to do anything interesting! You are converting something that is already a mass to a mass (and length to length, time to time) here, which will not even use the equivalency.